packages/zoe/src/contractSupport/zoeHelpers.js

```diff
@@ -146,48 +146,45 @@ export const makeZoeHelpers = zcf => {
       zcf.complete(handles);
       return defaultAcceptanceMsg;
     },
+
     /**
-     * Make an invitation to submit an Offer to this contract. This
-     * invitation can be given to a client, granting them the ability to
-     * participate in the contract.
+     * Make an offerHook that wraps the provided `offerHook`, to first
+     * check the submitted offer against an `expected` record that says
+     * what shape of proposal is acceptable.
      *
-     * If the "expected" option is provided, it should be an ExpectedRecord.
-     * This is like a Proposal, but the amounts in 'want' and 'give' should be null,
-     * and the 'exit' should have a choice but the contents should be null.
+     * This ExpectedRecord is like a Proposal, but the amounts in 'want'
+     * and 'give' should be null, and the 'exit' should have a choice but
```

Comment:
I'm having trouble interpreting "The 'exit' should have a choice". Would this say what you meant: "The exit clause should specify a rule with null contents."

Reply:
Good suggestion. Done.